.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         sudo apt-get install git g++-9 build-essential bison flex \
           libreadline-dev gawk tcl-dev libffi-dev git graphviz xdot \
           pkg-config libboost-system-dev libboost-python-dev \
-          libboost-filesystem-dev zlib1g-dev clang-format-5.0
+          libboost-filesystem-dev zlib1g-dev clang-format
```

Comment:
Do not use an unpinned clang-format. It means that different folks will run with different versions and they will tik-tok.

Reply:
mmh, the actual formatting information should come from the `.clang-format` file.
With a very specific .clang-format file I never had any problem.

Anyway, clang-format-11 is the current one. Should we pin to that ?

Reply:
Probably `clang-format-10`, as that at least supports Ubuntu LTS (18), see https://packages.ubuntu.com/search?keywords=clang-format

Reply:
Other good option would be `clang-format-8` which is on Ubuntu 16/18/20.

Reply:
Switched to clang-format-8.